agentassist/src/app/pages/dashboard/agent-feedback/agent-feedback.component.ts

```diff
@@ -259,6 +259,9 @@ export class AgentFeedbackComponent implements OnInit {
       if (response && response?.result?.length > 0) {
         this.feedbackSkip = this.feedbackSkip + 1;
         this.feedbackhasMore = response.hasMore;
+        response.result?.forEach(element => {
+          element.agentDetails = element.agentDetails ? JSON.parse(element.agentDetails) : {}
```

Comment:
@saradadh-Kore Write a try catch block for parsing as you are not stringifying it.

Reply:
resolved the comments